app/src/main/java/com/calculator/MainActivity.java

```diff
@@ -1,16 +1,480 @@
 package com.calculator;
 
+import androidx.annotation.Nullable;
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Switch;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public class MainActivity extends AppCompatActivity {
 
+    private static ArrayList<Character> OPERATORS = new ArrayList<>
+            (Arrays.asList('*', '/', '-', '+'));
+    private static int RIGHT_DIRECTION = 1;
+    private static int LEFT_DIRECTION = -1;
+    private int ACTIVITY2 = 123;
+
+    private static String syntaxError = "Syntax Error";
+    String result = "";
+    String values = "";
+
+    TextView label_values;
+    TextView label_result;
+    Button btn_number1;
```

Comment:
Try to declare this buttons in a group of buttons for example Button btn_number1_dev, btn_number2_dev, .... and Button btn_multiplication_adv, btn_division_adv, btn_equals_adv ...